test/unit_tests/hoc_python/test_reference_counts.py

```diff
@@ -0,0 +1,281 @@
+import sys
+import neuron
+from neuron import h
+
+
+def test_section():
+    section = h.Section(name='section')
+    assert sys.getrefcount(section) == 2
```

Comment:
Very nice. The following tedious proposition:
```suggestion
    base_refcount = 2
    assert sys.getrefcount(section) == base_refcount
```
Here it's pointless. It does protect a little against there currently being `2` references (when I can only see `1`).

Reply:
the extra one is the call to `sys.getrefcount`, I'll make a comment.
I thought about having a `base_refcount` like variable, so that's 2 of us.
I'll make the change.